test/Strings/rlexe.xml

```diff
@@ -230,4 +230,11 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>unicode_toUpperCase_toLowerCase.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>exclude_win7</tags>
```

Comment:
> Does the WinAPI you are using not exist in 7?

@farzonl , Windows 7 won't support [unicode planes](https://en.wikipedia.org/wiki/Plane_(Unicode)) that were added after Windows 7 was released. So the behavior of these test cases will be different (or will fail). 

> Does this bug need to be reopened against the linux branch

For cross platform effort, we will have to use some kind of [ICU](http://site.icu-project.org/) compliant library which will take care of these things. See #212 that has similar symptoms for `Intl` objects on Windows 7.